__tests__/ExpensiMark-test.js

```diff
@@ -283,3 +283,26 @@ test('Test general email link with various styles', () => {
 
     expect(parser.replace(testString)).toBe(resultString);
 });
+
+test('Test link with brackets', () => {
```

Comment:
I feel like a better name could be `'Test markdown and url links with inconsistent starting and closing parens'`.

Why? This subset of tests is unique because it's testing adding extra `(` and `)` at the beginning and end of links coming from pure urls & from markdown links.

I'm open to other names, but I think making it a bit more specific could be nice, since we already have url tests named `'Test a url ending with a closing parentheses autolinks correctly'` and `'Test markdown style link with various styles'`. If we make this test name super specific, this justifies why these test are in their own section.

Reply:
Surely, I agree.